src/test/java/io/neonbee/endpoint/metrics/NeonBeeMetricsIt.java

```diff
@@ -0,0 +1,117 @@
+package io.neonbee.endpoint.metrics;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.nio.file.Path;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.neonbee.NeonBee;
+import io.neonbee.NeonBeeOptions;
+import io.neonbee.config.EndpointConfig;
+import io.neonbee.endpoint.Endpoint;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpClientRequest;
+import io.vertx.core.http.HttpClientResponse;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.junit5.Checkpoint;
+import io.vertx.junit5.Timeout;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import io.vertx.micrometer.backends.BackendRegistries;
+
+@ExtendWith(VertxExtension.class)
+@SuppressWarnings("PMD.AvoidUnnecessaryTestClassesModifier")
+public class NeonBeeMetricsIt {
```

Comment:
What does "It" mean? IntegrationTest? We don't have used this pattern so far. When we want to introduce it, we should discuss it on Committer Meeting.

Reply:
Yes it stands for IntegrationTest.